### PR TITLE
TELCODOCS-1541 Documenting setting RCU to normal on start up

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1541]: Doc: Set RCU Normal

Version(s): 4.14 and main

Issue: https://issues.redhat.com/browse/TELCODOCS-1541

Link to docs preview: https://64613--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.html#ztp-setting-rcu-normal_sno-configure-for-vdu

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ztp-sno-du-setting-rcu-normal.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-setting-rcu-normal_{context}"]
+= Setting rcu_normal
+
+The following `MachineConfig` CR configures the system to set `rcu_normal` to 1 after the system has finished startup. This improves kernel latency for vDU applications.
```
Comment (ShaunaDiaz) [inline modules/ztp-sno-du-setting-rcu-normal.adoc:9]:
```suggestion
The following `MachineConfig` custom resource (CR) configures the system to set `rcu_normal` to 1 after the system has finished startup. This improves kernel latency for vDU applications.
```
Do you need to explain the "1" value? Not sure if it's a state or count and if your readers should know.

File: modules/ztp-sno-du-setting-rcu-normal.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+.Recommended configuration for disabling of `rcu_expedited` after node has finished startup (08-set-rcu-normal-master.yaml)
```
Comment (ShaunaDiaz) [inline modules/ztp-sno-du-setting-rcu-normal.adoc:11]:
```suggestion
.Recommended configuration for disabling `rcu_expedited` after the node has finished startup  in  the`08-set-rcu-normal-master.yaml` file
```

**Review by ShaunaDiaz — APPROVED** [review]
Looks great! Just a couple of nitty suggestions.

File: modules/ztp-sno-du-setting-rcu-normal.adoc (hunk 1 above)
Comment (kquinn1204) [reply modules/ztp-sno-du-setting-rcu-normal.adoc:9]: I personally don't think we need to get into it, it is detailed in the linked in 08-set-rcu-normal-master.yaml which I think is enough as it gives a description of what it does.

File: modules/ztp-sno-du-setting-rcu-normal.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-setting-rcu-normal_{context}"]
+= Setting rcu_normal
+
+The following `MachineConfig` CR configures the system to set `rcu_normal` to 1 after the system has finished startup. This improves kernel latency for vDU applications.
+
+.Recommended configuration for disabling `rcu_expedited` after the node has finished startup in  the`08-set-rcu-normal-master.yaml` file
```
Comment (jab-rh) [inline modules/ztp-sno-du-setting-rcu-normal.adoc:11]:
```suggestion
.Recommended configuration for disabling `rcu_expedited` after the node has finished startup in  the `08-set-rcu-normal-master.yaml` file
```
missing space

**Review by jab-rh — COMMENTED** [review]
@kquinn1204, I found a missing space.

**Comment by kquinn1204** [comment]
@jab-rh thanks for spotting that should be good to merge now.

**Pull request merged**